src/Http/Place/ImportPlaceRequestHandler.php

```diff
@@ -283,4 +263,26 @@ public function handle(ServerRequestInterface $request): ResponseInterface
         ];
         return new JsonResponse($responseBody, $responseStatus);
     }
+
+    public function guardDuplicatePlace(Place $place): void
+    {
+        if (! $this->preventDuplicatePlaces) {
+            return;
+        }
+
+        try {
+            $duplicatePlaceId = $this->lookupDuplicatePlace->getDuplicatePlaceUri($place);
+            if ($duplicatePlaceId !== null) {
+                throw ApiProblem::statusConflict(
+                    MultipleDuplicatePlacesFound::ERROR_MSG,
+                    ['query' => $duplicatePlaceId]
```

Comment:
When there is only one duplicate place, is it then still a query?

Reply:
What would you prefer as the param name?

Reply:
Something like the parameter and method name, maybe`duplicatePlaceUri`?

Reply:
changed it